addons/pos_restaurant/static/src/overrides/components/payment_screen/payment_screen.js

```diff
@@ -16,6 +16,11 @@ patch(PaymentScreen.prototype, {
         }
         return super.nextScreen;
     },
+    get amountPerGuest() {
```

Comment:
this same function appears in multiple places. Find a way to not repeat it. Maybe put it on the order model

Reply:
The implementation of the `amountPerGuest` function in the numpad.js depends on the state of the buffer to make it responsive to input, which differs from the one in the payment_screen.

Reply:
That should not be a problem. Look [here](https://github.com/odoo/odoo/blob/master/addons/point_of_sale/static/src/app/utils/input_popups/number_popup.js#L15) as an example